string-handling/string-characteristics/c-sharp/case_conversion.cs

```diff
@@ -0,0 +1,41 @@
+/*
+Isaac Computer Science
+Usage licensed under the Open Government Licence v3.0
+Note: This file is designed to be copied out and compiled on your machine.
+In order for it to compile properly you need to ensure that the project name is the same as the "namespace" in this file.
+To run this file you need to:
+1. Copy the contents
+2. Paste them into the C# IDE of your choice (Visual Studio, for example)
+3. Change the namespace to match your project (if neccesary)
+4. Compile the program
+5. Run the program
+*/
+
+using System;
+
+namespace IsaacCodeSamples
+{
+
+    class StringCharacteristics
+    {
+
+        // The Main method is the entry point for all C# programs
+        public static void Main() {
+            string letter = "B";
+            char lowerCase = ConvertToLower(letter);
+            Console.WriteLine(lowerCase);
+        }
+
+
+        // Demonstrates how to use ASCII codes to convert letter case
+        public static char ConvertToLower(string letter) {
```

Comment:
Why is the parameter not also a char?

Reply:
It will be now! But I will also need to change the function call.